modules/nf-core/sentieon/tnscope/tests/main.nf.test

```diff
@@ -0,0 +1,181 @@
+                    ]
+                input[3] = [[:],[]]
+                input[4] = [[:],[]]
+                input[5] = [[:],[]]
+                input[6] = [[:],[]]
+                input[7] = [[:],[]]
+                input[8] = [[:],[]]
+                """
+            }
+        }
+        then {
+            assert process.success 
+            assertAll(
+                { assert snapshot(
+                    path(process.out.vcf[0][1]).vcf.summary,
+                    file(process.out.index[0][1]).name,
+                    process.out.versions,
+                    path(process.out.versions[0]).yaml
+                    ).match() }
+            )
+        }
+    }
+
+    test("tumour normal - dbsnp") {
+
+        when {
+            params {
+                module_args  = ''
+                module_args2 = '--tumor_sample tumour --normal_sample normal'
+            }
+            process {
+                """
+                input[0] = NORMAL_ADDORREPLACEREADGROUPS.out.bam
+                    .join( NORMAL_ADDORREPLACEREADGROUPS.out.bai)
+                    .join( TUMOUR_ADDORREPLACEREADGROUPS.out.bam)
+                    .join( TUMOUR_ADDORREPLACEREADGROUPS.out.bai)
+                    .map{
+                    meta,normal_bam,normal_bai,tumour_bam,tumour_bai->
+                    [[id:'tumour'],[normal_bam,tumour_bam],[normal_bai,tumour_bai],[]]
+                    }
+                input[1] = [ [id:'test'],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta', checkIfExists:true)
+                    ]
+                input[2] = [ [id:'test'],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai', checkIfExists:true)
+                    ]
+                input[3] = [
+                    [id:'dbsnp'],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/germlineresources/dbsnp_138.hg38.vcf.gz',checkIfExists:true)
+                ]
+                input[4] = [
+                    [id:'dbsnp'],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/germlineresources/dbsnp_138.hg38.vcf.gz.tbi',checkIfExists:true)
+                ]                
+                input[5] = [[:],[]]
+                input[6] = [[:],[]]
+                input[7] = [[:],[]]
+                input[8] = [[:],[]]
+
+                """
+            }
+        }
+        then {
+            assert process.success
+            assertAll(
+                { assert snapshot(
+                    path(process.out.vcf[0][1]).vcf.summary,
+                    file(process.out.index[0][1]).name,
+                    process.out.versions,
+                    path(process.out.versions[0]).yaml
+                    ).match() }
+            )
+        }
+    }
+
+    test("tumour normal - no pon - stub") {
```

Comment:
"pon" could be set to "PON" to make it more evident that this is for Panel Of Normal 😉